src/coreComponents/physicsSolvers/contact/ContactFields.hpp

```diff
@@ -52,7 +52,7 @@ struct FractureState
 DECLARE_FIELD( penalty,
                "penalty",
```

Comment:
`contactPenalty` ?

Reply:
This penalty field is used in the ALM. Each element has its own penalty value, which can vary. Indeed the penalty is used to update the Lagrange multipliers and is adaptive to accelerate convergence.